### ISSUE TITLE
Deceptive Default browser setting.

### ISSUE THREAD

**Issue by consolecable** [brave/brave-browser#20489]
When the browser starts of the first time Brave tries to hide the fact that it is changing my default browser buy camouflaging the tick box ( with small fonts and light colours).

The is exactly the kind of deceptive behaviour that is ruining the internet and I would hope to avoid by by using an alternative browser like Brave.

I am not going to be using Brave or any other product that does this. Thanks for letting me know early on that you favour deceptive UI experiences. I will go out of my way to warn people about Brave.

**Comment by rebron** [comment]
@consolecable Can you show a screenshot of what you're seeing? Also please fill out the issue template so we know what OS you're using and other specifics.

We don't do deceptive UI experiences here. There very well may be a bug with the default browser setting dialog, so we would like to see what that looks like.

**Comment by consolecable** [comment]
Also, just to add the tick box is ticket by default. 

Even if it was easy to see I would consider this deceptive.

 Some user interaction should be required other than clicking OK. An option to ask me later would also help.

Here is the image when I untick it.

![image](https://user-images.githubusercontent.com/46242238/149758707-39070e18-a2cb-402c-b9be-f2ca32d81cd7.png)

**Comment by EriKWDev** [comment]
![image](https://user-images.githubusercontent.com/19771356/150680700-321841c5-ca1b-4059-b6ab-6a604f5dfa37.png)

Same on fedora first install. Please untick by default...

**Comment by EriKWDev** [comment]
Here (I believe) is the line that sets the checkbox to true:

https://github.com/brave/brave-core/blob/424c857a70b692360db3ad9c0caa2c722bac225e/chromium_src/chrome/browser/ui/views/first_run_dialog.cc#L105

**Comment by EriKWDev** [comment]
"We don't do deceptive UI experiences here. "

**Comment by EriKWDev** [comment]
I understand that a free browser needs all the help it can to spread, but not being honest in telling us that that is why this feature is on by default is deceptive in my book.

Will this ever change or do you plan on not commenting further?

**Comment by perpetual-hydrofoil** [comment]
How about eliminating the checkbox entirely:

```

Would you like to configure Brave to automatically open web pages by default?

Cancel                        Set Default Browser

```

**Comment by EriKWDev** [comment]
While that would sort of be a step in the right direction, I feel like it's still going around the fundamental issue I have with prompting the user like this right after install.

I don't know whether I want your browser as default before I've even tried it, and making it so easy to accidentally make it my default is just not cool.

In my book it, would make much more sense to have the question prompted first after having used the browser a couple of times so that the user doesn't make a decision they didn't want to make.

That's what's deceptful about the prompt in its current state and in the one you propose as well.

The user's obviously already invested in trying out brave's technology by downloading it in the first place. Do you really want this to be the face of your browser once it's run for the first time? A prompt using an obviously deceptful way to make more users accidentally put it as default?

Having the "make brave default" button to the right instead of the left is still a nudge that is a bit naughty in my book, but I guess it's better than not labeling the prompt correctly and having it checked by default...

**Comment by EriKWDev** [comment]
I also take issue with the lack of comment on this issue and transparency regarding this prompt.

The very first response in this thread was  "we don't do deceptive user experiences here" while the issue at hand has been completely ignored.

I proposed an explanation as to why it is that this prompt is needed, but you haven't commented on it so it just feels very strange..

**Comment by rebron** [comment]
Closing as fixed in 1.47.171 https://github.com/brave/brave-browser/issues/26378#issuecomment-1366905623
